src/workloads/scale/LargeScaleModel.yml

```diff
@@ -0,0 +1,469 @@
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: updateOne
+          Filter: {first : first}
+          Update: {$set: {second: *rand_1k_int}}
+    GlobalRate: *HotDocumentWrites
+
+- Name: HotCollectionLoader
+  Type: Loader
+  TrackProportion: *TrackProportion
+  Threads: 1
+  Phases:
+  - {Nop: true}
+  - Repeat: 1
+    Database: *HotCollectionDB
+    CollectionCount: 1
+    Threads: 1
+    DocumentCount: 1
+    BatchSize: 1
+    Document:
+      first: first
+      second: *rand_1k_int
+  - {Nop: true}
+
+- Name: HotCollectionUpdater
+  Type: CrudActor
+  TrackProportion: *TrackProportion
+  Threads: 40
+  Database: *HotCollectionDB
+  Phases:
+  - {Nop: true}
+  - {Nop: true}
+  - Duration: *Duration
+    Collection: Collection0
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: insertOne
+          Document: { a : *rand_1k_int}
+    GlobalRate: *HotCollectionWrites
+
+- Name: HotCollectionDeleter
+  Type: Deleter
+  Threads: 40
+  Database: *HotCollectionDB
+  Phases:
+  - {Nop: true}
+  - {Nop: true}
+  - Duration: *Duration
+    Collection: Collection0
+    GlobalRate: 1000 per 1 second
+
+# Create the cold database only.  For the hot database, we'll
+# use the rolling collection or the longlived collection, as they
+# are actively being updated and queried.
+- Name: ScannerLoader
+  Type: Loader
+  TrackProportion: *TrackProportion
+  # Note: we've cut down on the number of threads, as this load
+  # can sometimes causes failures.
+  Threads: 5
+  Phases:
+    # Each collection is ~100M.  Assuming N is equal to ScannerColdDBGigabytes.
+    # Each thread has N collections, and with 10 threads, that's
+    # 10*N collections, or N gigabytes of space.
+  - Repeat: 1
+    Database: *ScannerColdDB
+    CollectionCount: *ScannerColdDBGigabytes
+    Threads: 1
+    DocumentCount: 50000
+    BatchSize: 1000
+    Document:
+      a0: *rand_100b_string
+      a1: *rand_100b_string
+      a2: *rand_100b_string
+      a3: *rand_100b_string
+      a4: *rand_100b_string
+      a5: *rand_100b_string
+      a6: *rand_100b_string
+      a7: *rand_100b_string
+      a8: *rand_100b_string
+      a9: *rand_100b_string
+      a10: *rand_100b_string
+      a11: *rand_100b_string
+      a12: *rand_100b_string
+      a13: *rand_100b_string
+      a14: *rand_100b_string
+      a15: *rand_100b_string
+      a16: *rand_100b_string
+      a17: *rand_100b_string
+      a18: *rand_100b_string
+      a19: *rand_100b_string
+  - {Nop: true}
+  - {Nop: true}
+
+#- Name: CountScanner
+#  Type: CollectionScanner
+#  Threads: 1
+#  Database: *ScannerColdDB
+#  CollectionCount: 60
+#  Phases:
+#  - {Nop: true}
+#  - {Nop: true}
+#  - Duration: *Duration
+#    # Optionally skip the first iteration of the collection scanner
+#    SkipFirstLoop: true
+#    GlobalRate: 1 per 10 seconds
+#    ScanType: count
```

Comment:
I'd either remove these actors or add an additional comment saying why they're commented-out / link to a ticket/TODO to uncomment them.

Reply:
I'll remove them (I just had them there to remind me how to do these things).